nuxeo-management-rest-api/src/main/resources/OSGI-INF/rest-management-marshallers-contrib.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component name="org.nuxeo.rest.management.marshallers">
+<component name="org.nuxeo.rest.management.marshallers.distribution">
```

Comment:
What is the signification of `distribution` here?

Reply:
This is a trace of when I made separate contributions per endpoints. One of them was distribution. Took it down !